enhancements/service-ca-bundle-injection-for-webhooks.md

```diff
@@ -0,0 +1,110 @@
+---
+title: service-ca-bundle-injection-for-webhooks
+authors:
+  - "@marun"
+reviewers:
+  - "@deads2k"
+  - "@sttts"
+  - "@stlaz"
+approvers:
+  - "@deads2k"
+  - "@sttts"
+creation-date: 2020-01-23
+last-updated: 2020-01-23
+status: implementable
+see-also:
+  - https://github.com/openshift/service-ca-operator/pull/79 (Implementation)
+replaces:
+superseded-by:
+---
+
+# Support Service CA Bundle Injection for Webhooks
+
+## Release Signoff Checklist
+
+- [ ] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Summary
+
+Webhooks can secure their endpoints automatically with a serving cert
```

Comment:
Be more precise throughout the doc that this is about admission webhooks. We have a number of webhooks in kube meanwhile.

Reply:
Done